### ISSUE TITLE
IPFS and Config

### ISSUE THREAD

**Issue by hobbydev71** [dabit3/ipfs-pinata-example#1]
It's simple and clear code.
But I'm not sure which package is used for IPFS and how to config.

**Comment by abhinay-gh** [comment]
Yeah the packages are not mentioned. Pls add details of what packages to use.

**Comment by mythpay** [comment]
You can use this package: ipfs-http-client
